### PR TITLE
Homework 2 polindrom

### PR DESCRIPTION
**Pull request by Denis-Pikus**
I have done task @Palindrome"

### CODE REVIEW

**Review by KateRin1103 — APPROVED** [review]

File: IntensiveJavaGuru/src/main/java/dpdev/homework_2_palindrome/PalindromeDemo.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package dpdev.homework_2_palindrome;
+
+import dpdev.homework_2_palindrome.service.Palindrome;
+
+public class PalindromeDemo {
+    public static void main(String[] args) {
+        Palindrome palindrome = new Palindrome();
+        System.out.println(palindrome.isPalindrome("А роза упала на лапу Азора"));
```
Comment (zhdanok) [inline IntensiveJavaGuru/src/main/java/dpdev/homework_2_palindrome/PalindromeDemo.java:8]: I think it's better use logger their

**Review by zhdanok — APPROVED** [review]
lgfm

**Comment by Denis-Pikus** [comment]
merge pull equest

**Comment by Denis-Pikus** [comment]
create pull request

**Pull request merged**